lib/v1/statuses/index.js

```diff
@@ -0,0 +1 @@
+module.exports.create = { request: require('./create.request') }
```

Comment:
Keep EOF at the end of the files.
```suggestion
module.exports.create = { request: require('./create.request') }

```